src/Sfa.Tl.Matching.Application.UnitTests/Services/ProviderFeedback/When_ProviderFeedbackService_Is_Called_To_Send_Provider_Quarterly_Update_Emails_With_No_Venues.cs

```diff
@@ -0,0 +1,321 @@
+            history.Id.Should().Be(1);
+            history.Status.Should().Be(ProviderFeedbackRequestStatus.Complete.ToString());
+            history.ProviderCount.Should().Be(1);
+            history.ModifiedBy.Should().Be("TestUser");
+        }
+
+        [Fact]
+        public void Then_EmailService_SendEmail_Is_Called_Exactly_Once()
+        {
+            _emailService
+                .Received(1)
+                .SendEmail(Arg.Any<string>(), Arg.Any<string>(), Arg.Any<string>(), Arg.Any<IDictionary<string, string>>(), Arg.Any<string>());
+        }
+
+        [Fact]
+        public void Then_EmailService_SendEmail_Is_Called_With_TemplateName()
+        {
+            _emailService
+                .Received(1)
+                .SendEmail(Arg.Is<string>(
+                        templateName => templateName == "ProviderQuarterlyUpdate"),
+                    Arg.Any<string>(),
+                    Arg.Any<string>(),
+                    Arg.Any<IDictionary<string, string>>(),
+                    Arg.Any<string>());
+        }
+
+        [Fact]
+        public void Then_EmailService_SendEmail_Is_Called_With_ToAddress()
+        {
+            _emailService
+                .Received(1)
+                .SendEmail(Arg.Any<string>(),
+                    Arg.Is<string>(
+                        toAddress => toAddress == "primary.contact@provider.co.uk"),
+                    Arg.Any<string>(),
+                    Arg.Any<IDictionary<string, string>>(),
+                    Arg.Any<string>());
+        }
+
+        [Fact]
+        public void Then_EmailService_SendEmail_Is_Called_With_Subject()
+        {
+            _emailService
+                .Received(1)
+                .SendEmail(Arg.Any<string>(),
+                    Arg.Any<string>(),
+                    Arg.Is<string>(
+                        subject => subject == "Industry Placement Matching Provider Update"),
+                    Arg.Any<IDictionary<string, string>>(),
+                    Arg.Any<string>());
+        }
+
+        [Fact]
+        public void Then_EmailService_SendEmail_Is_Called_With_ReplyToAddress()
+        {
+            _emailService
+                .Received(1)
+                .SendEmail(Arg.Any<string>(),
+                    Arg.Any<string>(),
+                    Arg.Any<string>(),
+                    Arg.Any<IDictionary<string, string>>(),
+                    Arg.Is<string>(
+                        replyToAddress => replyToAddress == ""));
+        }
+
+        [Fact]
+        public void Then_EmailService_SendEmail_Is_Called_With_ProviderName_Token()
+        {
+            _emailService
+                .Received(1)
+                .SendEmail(Arg.Any<string>(),
+                    Arg.Any<string>(),
+                    Arg.Any<string>(),
+                    Arg.Is<IDictionary<string, string>>(
+                        tokens => tokens.ContainsKey("provider_name")
```

Comment:
extract Token in to a variable and then use a standard function to assert each token values